acid-view/app/Main.hs

```diff
@@ -0,0 +1,23 @@
+module Main where
+
+import AcidView (loadState, printDB)
+import CmdParse (CmdOptions(..), parsedCMDOptions)
+import Data.ByteString.Lazy.Char8 (unpack)
+
+import Ariadne.Wallet.Cardano.Kernel.DB.AcidState (Snapshot(..), defDB)
+import Data.Acid (query)
+
+main :: IO ()
+main = do
+  CmdOptions{..} <- parsedCMDOptions
+  stateE <- loadState path defDB position
+  case stateE of
+    Left err -> print err
+    Right (acidDB,eventTag) -> do
+      db <- liftIO $ query acidDB Snapshot
+      printDB output fileToSave db
+      let appliedMethod = "\nLast applied method: " ++ unpack eventTag ++ "\n"

```

Comment:
We prefer to use `<>` instead of `++`. Also, there seems to be `putText` in `universum` (IIRC), so you don't have to use `unpack`.

Reply:
Ok, thanks